mule-maven-plugin/src/main/java/org/mule/tools/maven/mojo/ProcessResourcesMojo.java

```diff
@@ -29,11 +29,11 @@
 
   @Override
   public void doExecute() throws MojoExecutionException, MojoFailureException {
-    try {
-      getResourcesContentProcessor().process(resourcesContent);
-    } catch (IllegalArgumentException | IOException e) {
-      throw new MojoFailureException("Fail to process resources", e);
-    }
+    // try {
```

Comment:
Could you please remove the commented code if it is not necessary anymore?

Reply:
nope my bad
I was already commented out but I did not remove it I'll do it now

Reply:
done